### DISCUSSION TITLE
Custom Dialog Container

### DISCUSSION THREAD

**Discussion by razorxan** [radix-ui/primitives#961] [Help] [answered]
As far as i can see, dialog are by default rendered in the body tag. Is there a way to render a dialog in a custom container?

**Comment by benoitgrelard** [comment, marked as answer]
We've been experimenting with exposing the `Portal` part in `Dialog` to enable that kind of control.
Take a look at this: https://github.com/radix-ui/primitives/issues/760#issuecomment-952102287

It's available in the latest release candidates as linked in that comment.

**Reply by razorxan** [reply]
ended up uninstall the library but thanks for the reply

**Reply by benoitgrelard** [reply]
Hey @razorxan, can I ask what we could have done to make it work for you?
